source/features/pr-jump-to-first-unviewed-file.tsx

```diff
@@ -0,0 +1,34 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function jumpToFirstUnviewed(): void {
+	const firstNotView = select('.js-reviewed-checkbox:not([checked])')!;
+	if (!firstNotView) {
+		// Scroll to page end
+		window.scrollTo(0, document.body.scrollHeight);
```

Comment:
.

Use the same solution you use in #3242

You could even wrap it in a `goToFragment()` function

Reply:
Not sure what you are referring to.

Reply:
Sorry I got confused with the line below. How come this is only `location.replace` while the other one also has `scrollIntoView`?

Reply:
Here it correctly jumps to the correct place. There is causes an unexpected jump.